process-automation/latest/model/data-models.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Data Models
+---
+
+A data model allows you to represent complex objects your application is handling. It allows you to define the standard data structure you will use in your application using JSON schema. A data model is useful when you want to query an internal or external service that has a specific structure.
+The data model editor allows you to visually create the JSON schema and the JSON editor allows you to see the JSON code of the schema. When you use a data model it provides auto-completion in your scripts and expressions. You can also map a data model with your process variables and connector properties. You can use the Upload button to upload a previously defined JSON schema that you have downloaded from Process Automation.
+
+These instructions describe how to make a data model for a specific process but you can create a data model for use at a global level where the same data model properties can be used across multiple processes. To create a global data model Click **New** > **Data model** from the home screen. Alternatively you can transform your data model into a global one by selecting the three dots on the right hand side of your open data model and selecting **Turn global**.  
+
+**Note:** If you transform your data model into a global one you are unable to revert it.
+
+## Properties
+
+The properties of a data model are:
+
+| Property | Description |
+| -------- | ----------- |
+| Data model name | *Required.* The name of the data model. Must be in lowercase and between 1 and 26 characters in length. Alphanumeric characters and hyphens are allowed, however the name must begin with a letter and end alphanumerically, for example 'Person'. |
+| Type | *Required.* Use the Type field to select the JSON schema type you want to use, for example `object`. You can't select more than one JSON schema type from the dropdown list. You can select more than one type from the Modelling app, Enumeration, and Composition sections. |
```

Comment:
You can select more than one type from the JSON schema types (for example integer and string) It is supported unless it is a very unlikely scenario

Reply:
I have changed it to say you can select more than one except for ones in the Modelling App.